PowerEditor/installer/nativeLang/serbian.xml

```diff
@@ -5,7 +5,7 @@
 		- Radoš Milićev (https://github.com/rammba)
 -->
 <NotepadPlus>
-	<Native-Langue name="Srpski(Serbian)" filename="serbian.xml"  version="8.7.2">
```

Comment:
In language dropdown in the preferences dialog only label `Srpski` is present, so I omitted english name. Is this ok?